tests/rptest/perf/openmessaging_perf_test.py

```diff
@@ -25,23 +27,50 @@ def __init__(self, ctx):
               self).__init__(test_context=ctx, num_brokers=3)
 
     @cluster(num_nodes=6)
-    @parametrize(driver_idx="ACK_ALL_GROUP_LINGER_1MS_IDEM_MAX_IN_FLIGHT",
-                 workload_idx="RELEASE_CERT_SMOKE_LOAD_625k")
-    def test_perf(self, driver_idx, workload_idx):
-        """
-        This test is run as a part of nightly perf suite to detect
-        regressions.
-        """
-
-        # Make sure this is running in a dedicated environment as the perf
-        # run validator metrics are based on a production grade deployment.
-        # Check validator for specifics.
-        assert self.redpanda.dedicated_nodes
-
-        benchmark = OpenMessagingBenchmark(self._ctx, self.redpanda,
-                                           driver_idx, workload_idx)
+    def omb_test(self):
+        workload = {
+            "name": "CommonWorkload",
+            "topics": 1,
+            "partitions_per_topic": 100,
+            "subscriptions_per_topic": 1,
+            "consumer_per_subscription": 25,
+            "producers_per_topic": 25,
+            "producer_rate": 175_000,
+            "message_size": 1024,
+            "payload_file": "payload/payload-1Kb.data",
+            "consumer_backlog_size_GB": 0,
+            "test_duration_minutes": 5,
+            "warmup_duration_minutes": 1,
+        }
+        driver = {
+            "name": "CommonWorkloadDriver",
+            "replication_factor": 3,
+            "request_timeout": 300000,
+            "producer_config": {
+                "enable.idempotence": "true",
+                "acks": "all",
+                "linger.ms": 1,
+                "max.in.flight.requests.per.connection": 5,
+                "batch.size": 131072,
```

Comment:
Lets set this to 16KB? It's a lot more realisitic and avoids us running in larger forced batch sizes.

Reply:
Nice catch, switching it to the default of `16384`.